src/transformers/tools/agents.py

```diff
@@ -259,6 +259,44 @@ def load_tools_if_needed(self, remote=False):
                 self._tools[name] = load_tool(task_or_repo_id, remote=_remote)
 
 
+class AgentError(Exception):
```

Comment:
Should this be a subclass of `RuntimeError`?

Reply:
Thank you! This is changed.